src/basic/glyph-util.c

```diff
@@ -137,5 +139,7 @@ const char *special_glyph(SpecialGlyph code) {
                 return NULL;
 
         assert(code < _SPECIAL_GLYPH_MAX);
+        if (force_utf)
```

Comment:
instead do `force_utf || emoji_enabled()` below

Reply:
> instead do `force_utf || emoji_enabled()` below

I don't quite understand this yet. I'll look at it more and push the modification sir.

Reply:
What I can make of it so far is ```force_utf || emoji_enabled() ``` will be true if emojis are enabled or we're forcing utf8 or both, and will only be false if emojis are not enabled and we're not forcing utf8, which is exactly the expected behaviour.

I could change this 
```return draw_table[code >= _SPECIAL_GLYPH_FIRST_EMOJI ? emoji_enabled() : is_locale_utf8()][code]```

to
```return draw_table[force_utf || emoji_enabled()][code]```

but we lose this check: ```code >= _SPECIAL_GLYPH_FIRST_EMOJI ? emoji_enabled() : is_locale_utf8()```

What do you advise sir?

Reply:
```
return draw_table[force_utf || code >= _SPECIAL_GLYPH_FIRST_EMOJI ? emoji_enabled() : is_locale_utf8()][code]

```

Reply:
Keep the check - `force_utf || code >= _SPECIAL_GLYPH_FIRST_EMOJI ? emoji_enabled() : is_locale_utf8()`

Reply:
> Keep the check - `force_utf || code >= _SPECIAL_GLYPH_FIRST_EMOJI ? emoji_enabled() : is_locale_utf8()`

Looks like I said it first 😂😂😂😂
Let me modify and push sir.